blackhole_watcher.py

```diff
@@ -2,6 +2,359 @@
 from watchdog.observers import Observer
 from watchdog.events import FileSystemEventHandler
 from blackhole import on_created, getPath
+from fastapi import FastAPI, WebSocket, WebSocketDisconnect
+from fastapi.staticfiles import StaticFiles
+from fastapi.responses import FileResponse
+from pydantic import BaseModel
+from typing import List, Optional, Dict, Any, Tuple
+from shared.arr import Radarr, Sonarr
+from shared.websocket import WebSocketManager
+import uvicorn
+import os
+from fastapi.middleware.cors import CORSMiddleware
+from fastapi import HTTPException
+import requests
+
+app = FastAPI()
+
+# Add CORS middleware
+app.add_middleware(
+    CORSMiddleware,
+    allow_origins=["*"],
+    allow_credentials=True,
+    allow_methods=["*"],
+    allow_headers=["*"],
+)
+
+# Mount static files
+app.mount("/static", StaticFiles(directory="/app/static", html=True), name="static")
+
+# API routes must come before the catch-all route
+@app.get("/api/tmdb/{media_type}/{tmdb_id}/images")
+async def get_tmdb_images(media_type: str, tmdb_id: int):
+    api_key = os.getenv('TMDB_API_KEY')

```

Comment:
We have a system for getting and using environment variables in the code

Reply:
I didn’t see or know about that